internal/pkg/api/cloudstack.go

```diff
@@ -67,6 +68,33 @@ func WithCloudStackManagementServer(value string) CloudStackFiller {
 	}
 }
 
+func WithCloudStackAz(az anywherev1.CloudStackAvailabilityZone) CloudStackFiller {
+	return func(config CloudStackConfig) {
+		config.datacenterConfig.Spec.AvailabilityZones = append(config.datacenterConfig.Spec.AvailabilityZones, az)
+	}
+}
+
+func WithFirstCloudStackAz(az anywherev1.CloudStackAvailabilityZone) CloudStackFiller {
+	return func(config CloudStackConfig) {
+		config.datacenterConfig.Spec.AvailabilityZones[0] = az
```

Comment:
Same here, but what is the purpose of this? Instead of making it first cloudstack az, can we pass in index and attempt to set it at that spot? Otherwise if it is just for modifying one of them, I would just reset the cloudstack azs and re-add them.